vacation_calendar/templates/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+  height: 100vh;
+  display: grid;
+  grid-template-columns: 300px 1fr;
+  grid-template-rows: 60px 1fr;
+  font-family: "Avenir Next LT W04 Regular", sans-serif;
+  margin: 0;
+}
+h2 {
+  font-family: "Tw Cen MT W03 Medium Cond", sans-serif;
+  font-weight: 350;
+}
+
+/* Top Bar */
+.topbar {
+  background-color: #ffffff;
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+}
+
+.topbar-menu {
+  display: flex;
+  justify-content: space-between;
+  padding: 0px 5px;
+}
+
+.left,
+.right {
+  display: flex;
+  align-items: center;
+  padding: 0px 20px;
+}
+
+.logout-btn {
+  border: none;
+  text-decoration: none;
+  background: #ffdb45;
+  color: #222;
+  margin: 0;
+  text-align: center;
+  font-family: "Tw Cen MT W04 Medium", sans-serif;
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.logout-btn {
+  padding: 2px 16px;
+  font-size: 20px;
+}
+
+/* Left Side Bar */
+.leftside {
+  background-color: #292b2c;
+  grid-column: 1 / 2;
+  grid-row: 1 / 3;
+}
+
+.logo {
+  text-align: center;
+  height: 60px;
+  padding-top: 10px;
+}
+
+.leftside-menu {
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+ul {
```

Comment:
since this is not for a specific section, but rather controls the overall behavior of the page, I'd put these styles closer to the top